src/content/h1/tools/aether/readme.md

```diff
@@ -1,15 +1,219 @@
+
+Aether doesn't release the lightmap bitmap until you close the program. So if you get file in use errors when compiling your map, make sure Aether is closed.
+
+Thats it! lightmap done! Now that you have everything set up you can create quick iterations of your lighting without having to wait a day for lightmaps to run. 
+
+**Custom Lightmap UVs**
+Custom Lightmap UVs
+Example of poor UVs by Tool
+
+When viewing your custom lightmaps you may find that there are areas that aren't correctly lit, or where the shadows dont line up, etc. This is because Tool wasn't designed to make lightmap UVs suited to high resolution lightmaps, which results in small overlaps and lack of consideration for lightmap UV alignment, which would otherwise be unnoticable with Halos default lightmaps. To tackle this problem a custom lightmap UV importer has been developed for UserTool[2] so that designers have complete control over their lightmaps UVs and can now remove such problems.
+
+The recommended workflow for creating completely custom lightmap UVs:
+
+<figure>
+  <a href="Poor_UVs_Ex.jpg">
+    <img src="Poor_UVs_Ex.jpg" alt="Example of poor UVs by Tool"/>
+  </a>
+  <figcaption>
+    <p>Example of poor UVs by Tool</p>
+  </figcaption>
+</figure>
+
+1. Finalise your BSP.
+    It would be unwise to make custom lightmaps for each BSP iteration as it does take time to Re-UV a BSP so you don't want to be doing it over and over again. 
+2. Import your current lightmap UVs into your modelling program.
+    Export your current lightmaps with Aether and import them into your modelling program. As with the custom lightmap tutorial you should import them as seperate objects. 
+3. Re UV the lightmap meshes.
+     You can UV your meshes as you would UV any other mesh, however, DO NOT change the mesh in any other way as this could change the surface indicies of the mesh and it would not match those that are in the BSP.
+     Your UVs also have to be in the range of 0.0 - 1.0 on both axes. 
+4. Export your lightmap meshes to an obj file.
+     The OBJ file needs to at least have vertex position and texture coordinates to be valid. Normals is unnecessary but adding them shouldn't cause problems. 
+6. Import your new lightmap UVs into the BSP using the import-lightmap-uv command in UserTool.
+     The command requires two arguments. <bsp-file> which is the target BSP relative to the tags directory and <obj-file> which is the source OBJ relative to your data directory. 
+7. Do your custom lighting as normal, either using the tutorials method or your own. 
+
+If you only want to fix the problem areas created by Tool then you would do a quick custom lighting solution to highlight the problem areas and then do the above but only fixing those areas. 
+
+
 
 
-[dithering]: https://en.wikipedia.org/wiki/Dither
```

Comment:
Removing this will cause the dithering link in the page's introduction to no longer work.